app/lib/proxy_config_affecting_changes.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module ProxyConfigAffectingChanges
+  module ProxyRuleExtension
+    extend ActiveSupport::Concern
+
+    included do
+      include ProxyConfigAffectingChanges
+
+      after_commit :issue_proxy_affecting_change_events
+
+      def issue_proxy_affecting_change_events
+        case owner
+        when Proxy
+          issue_proxy_affecting_change_event(owner)
+        when BackendApi
+          owner.proxies.each(&method(:issue_proxy_affecting_change_event))
+        end
+      end
+    end
+  end
+
+  module ProxyExtension
+    extend ActiveSupport::Concern
+
+    PROXY_CONFIG_AFFECTING_ATTRIBUTES = %w[policies_config].freeze # TODO: add more attributes here
+
+    included do
+      include ProxyConfigAffectingChanges
+
+      after_commit :issue_proxy_affecting_change_events, on: :update
+
+      def issue_proxy_affecting_change_events
+        changes_attributes = previous_changes.keys
```

Comment:
So you can use `previously_changed?` https://github.com/3scale/porta/blob/d21f8d5e7a6dc740db6652e0604dcf0b639edb52/lib/previously_changed.rb

Which will be introduced in rails 5.0